products/windows.md

```diff
@@ -9,236 +9,259 @@ releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows
+    releaseLabel: "10 21H2 (W)"
     releaseDate: 2021-11-16
     support: 2023-06-13
     eol: 2023-06-13
     latest: 10.0.19044
     link: https://learn.microsoft.com/windows/release-health/release-information
 
--   releaseCycle: "11, version 21H2 (E)"
+-   releaseCycle: "11-21H2-E"
+    releaseLabel: "11 21H2 (E)"
     releaseDate: 2021-10-04
     support: 2024-10-08
     eol: 2024-10-08
     latest: 10.0.22000
     link: https://learn.microsoft.com/windows/release-health/windows11-release-information
 
--   releaseCycle: "11, version 21H2 (W)"
+-   releaseCycle: "11-21H2-W"
+    releaseLabel: "11 21H2 (W)"
     releaseDate: 2021-10-04
     support: 2023-10-10
     eol: 2023-10-10
     latest: 10.0.22000
     link: https://learn.microsoft.com/windows/release-health/windows11-release-information
 
--   releaseCycle: "10, version 21H1 (E)(W)"
+-   releaseCycle: "10-21H1"
+    releaseLabel: "10 21H1"
     releaseDate: 2021-05-18
     support: 2022-12-13
     eol: 2022-12-13
     latest: 10.0.19043
     link: https://learn.microsoft.com/windows/release-health/status-windows-10-21h1
 
--   releaseCycle: "10, version 20H2 (E)"
+-   releaseCycle: "10-20H2-E"
+    releaseLabel: "10 20H2 (E)"
     releaseDate: 2020-10-20
     support: 2023-05-09
     eol: 2023-05-09
     latest: 10.0.19042
     link: https://learn.microsoft.com/windows/release-health/status-windows-10-20h2
 
--   releaseCycle: "10, version 20H2 (W)"
+-   releaseCycle: "10-20H2-W"
+    releaseLabel: "10 20H2 (W)"
     releaseDate: 2020-10-20
     support: 2022-05-10
     eol: 2022-05-10
     latest: 10.0.19042
     link: https://learn.microsoft.com/windows/release-health/status-windows-10-20h2
 
--   releaseCycle: "10, version 2004 (E)(W)"
+-   releaseCycle: "10-2004"
+    releaseLabel: "10 2004"
     releaseDate: 2020-05-27
     support: 2021-12-14
     eol: 2021-12-14
     latest: 10.0.19041
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-version-2004-end-of-servicing
 
--   releaseCycle: "10, version 1909 (E)"
+-   releaseCycle: "10-1909-E"
+    releaseLabel: "10 1909 (E)"
     releaseDate: 2019-11-12
     support: 2022-05-10
     eol: 2022-05-10
     latest: 10.0.18363
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1909-enterprise-education-eos
 
--   releaseCycle: "10, version 1909 (W)"
+-   releaseCycle: "10-1909-W"
+    releaseLabel: "10 1909 (W)"
     releaseDate: 2019-11-12
     support: 2021-05-11
     eol: 2021-05-11
     latest: 10.0.18363
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1909-end-of-servicing
 
--   releaseCycle: "10, version 1903 (E)(W)"
+-   releaseCycle: "10-1903"
+    releaseLabel: "10 1903"
     releaseDate: 2019-08-29
     support: 2020-12-08
     eol: 2020-12-08
     latest: 10.0.18362
     link: https://learn.microsoft.com/lifecycle/announcements/windows-10-1903-end-of-servicing
 
--   releaseCycle: "10, version 1809"
+-   releaseCycle: "10-1809"
```

Comment:
Is it normal to have 10 1809, 10 1809 (E) and 10 1809 (W) ?

Reply:
Solved with 7a09b7d1df4efe13102d3ed2ad15420ea39799fd